### DISCUSSION TITLE
Is there a way to use this in a C# Project?

### DISCUSSION THREAD

**Discussion by DomDom3333** [CPJKU/madmom#504] [Q&A]
I have little to no experience with python and was hoping there is a way to use this (specifically the Beat detection) in a C# project. I tried IronPython but I'm having issues with packages as far as i can tell. Any info/help would be GREATLY appreciated.

**Comment by superbock** [comment]
I'm sorry, but I can't help with C# integration of madmom. Probably look for generic information on how to run Python code in C# projects.
